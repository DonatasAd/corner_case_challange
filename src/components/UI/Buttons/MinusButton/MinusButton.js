import React from 'react';
import BackArrow from '../../../../assets/minus-button.png';
import styles from './MinusButton.module.css';

export default function MinusButton(props) {
  return (
    <button {...props} className={styles.Button}>
      <img src={BackArrow} className={styles.Logo} alt='' />
    </button>
  );
}

MinusButton.defaultProps = {
  onClick: () => console.log('Minus button'),
};
