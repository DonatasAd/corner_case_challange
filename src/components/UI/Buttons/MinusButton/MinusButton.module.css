.Logo {
  height: 100%;
}

.Button {
  padding: 0;
  height: 1.7rem;
  background: transparent;
  box-shadow: 0px 0px 0px transparent;
  border: 0px solid transparent;
  text-shadow: 0px 0px 0px transparent;
}

.Button:hover {
  cursor: pointer;
  background: transparent;
  box-shadow: 0px 0px 0px transparent;
  border: 0px solid transparent;
  text-shadow: 0px 0px 0px transparent;
}

.Button:active {
  outline: none;
  border: none;
}

.Button:focus {
  outline: 0;
}
