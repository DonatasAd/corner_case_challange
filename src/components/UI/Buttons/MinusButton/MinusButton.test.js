import React from 'react';
import { shallow } from 'enzyme';
import MinusButton from './MinusButton';

describe('MinusButton', () => {
  it('should render correctly in "debug" mode', () => {
    const component = shallow(<MinusButton />);
    expect(component).toMatchSnapshot();
  });
});