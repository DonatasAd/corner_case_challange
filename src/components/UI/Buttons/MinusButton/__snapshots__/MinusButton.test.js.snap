// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MinusButton should render correctly in "debug" mode 1`] = `ShallowWrapper {}`;
